docs/spec_draft.md

```diff
@@ -2234,6 +2235,39 @@ hidden state.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.round_nearest_even
+
+### Semantics
+
+Performs element-wise round, rounding to nearest integer with ties towards even,
```

Comment:
@ghpvnist you can also include RoundOp in this, I have assigned that to you as well since both are very similar.

Reply:
Performs element-wise rounding towards the nearest integer, breaking ties towards the even integer, on the `operand` tensor and produces a `result` tensor.
How about this? wdyt?